### PR TITLE
819 add op cost calculation and explain fee parameters to wails pop ups callsc executesc transaction roll buysell

### PR DESCRIPTION
**Pull request by Thykof**

### CODE REVIEW

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 1]
```diff
@@ -31,6 +33,13 @@ export function CallSc(props: PromptRequestData) {
     Nickname,
   } = props;
 
+  const [operationCost, setOperationCost] = React.useState(0n);
+
+  React.useEffect(() => {
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:38]:
```suggestion
  useEffect(() => {
```

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import React from 'react';
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:1]:
```suggestion
import { useEffect } from 'react';
```

File: wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx [hunk 3]
```diff
@@ -79,30 +88,68 @@ export function CallSc(props: PromptRequestData) {
...
       )}
 
-      {Description && <hr className="h-0.25 bg-neutral opacity-40 w-full" />}
-
+      <div className="flex flex-col w-full h-fit">
+        <AccordionCategory
+          isChild={false}
+          iconOpen={<FiChevronDown />}
+          iconClose={<FiChevronUp />}
+          customClass="!p-0"
```
Comment (mazmassa) [inline wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:121]: Please, don't use overrides on class: `!p-0`

@pivilartisant remember we fixed that. Isn't it?

Comment (pivilartisant) [reply wails-frontend/src/pages/PasswordPromptHandler/SignSC/CallSc.tsx:121]: I guess, maybe it is another page that we fixed

**Comment by Thykof** [comment]
change branch name

**Pull request closed without merging**